projects/components/src/utils/test/widget-object/angular/angular-widget-object-element.ts

```diff
@@ -161,6 +192,13 @@ export class TestElement implements Iterable<TestElement> {
         return this.elements.map((el) => new TestElement([el], this.fixture));
     }
 
+    /**
+     * Maps over each element of this collection
+     */
+    map<U>(callbackfn: (el: TestElement, index: number, array: TestElement[]) => U): U[] {
```

Comment:
Total extra => is it possible for us to define 
`map = this.toArray().map` and then we'll get all the typescript info + the signature of map for free?